File: Java/6.27.java
/*
Write a program whose input is two integers and whose output is the two integers swapped. Place the values in an array, where x is position 0 and y is position 1.

Ex: If the input is:

3 8
then the output is:

8 3
*/

import java.util.Scanner;

public class LabProgram {
   
   /* Define your method here */
   public static void swapValues(int[] values) {
      int tempVal = values[0];
      values[0] = values[values.length - 1];
      values[values.length - 1] = tempVal;
   }
   
   public static void main(String[] args) {
      /* Type your code here. */
      Scanner scan = new Scanner(System.in);
      int[] userValues = new  int[2];
      
      for (int i = 0; i < userValues.length; i++) {
         userValues[i] = scan.nextInt();
      }
      
      swapValues(userValues);
      
      for (int i = 0; i < userValues.length; i++) {
         System.out.printf("%d ", userValues[i]);
      }
      System.out.println();
      
      
   }
}

File: CPP/6.27.cpp
/*
Write a program that removes all spaces from the given input.

Ex: If the input is:

Hello my name is John.
the output is:

HellomynameisJohn.
*/

#include <iostream>
using namespace std;

/* Define your function here */ 
string RemoveSpaces(string userString) {
   string newString;
   for (int i = 0; i < userString.size(); ++i) {
      if (userString[i] != ' ') {
         newString += userString[i];
      }
   }
   return newString;
}

int main() {
   /* Type your code here. Your code must call the function.  */
   string userStringInput;
   
   getline(cin, userStringInput);
   
   printf("%s\n", RemoveSpaces(userStringInput).c_str());

   return 0;
}
